README.md

```diff
@@ -71,3 +71,100 @@ Calls are made to Google Analytics regarding users' mfas and whether a password
+`EXTERNAL_GROUPS_SYNC_*` entries in the `local.env.dist` file.
+
+### How Many of What?
+
+You will need...
+
+* One Google Cloud Console Project.
+  * Example:  
+    `My IDPs External Groups Sync`
+* One Google Sheet per application that needs custom groups.
+  * Examples:  
+    `App A SSO Groups`  
+    `App B SSO Groups`
+* One Service Account (in that Project) per IDP that you want to sync the
+  custom groups into for that application.
+  * Examples:  
+    `IDP 1 groups for App A`  
+    `IDP 2 groups for App A`
+* One tab in each Google Sheet per IDP that you want to sync that application's
+  custom groups into.
+  * Examples:  
+    `idp1`  
+    `idp2`
+
+### Specific How-To Steps
+
+To do this...
+
+1. Use at least version 6.8.0 of ID Broker.
+2. Create a Google Sheet named for the application that needs the groups
+   (e.g. `App A SSO groups`).
+3. Create a tab (in that Google Sheet) named after the short/code name of your
+   IDP (e.g. `idp1`) with two columns: `email` and `groups`.
+   - To add groups for a specific user, put the user's (lowercase) email address
+     for that IDP in the `email` cell in their row.
+   - Only use one row per user.
+   - Put all of a user's desired groups in their `groups` cell, separated by
+     commas. Example: "ext-appa-managers, ext-appa-designers"
+   - Group names must begin with your chosen prefix and a dash
+     (e.g. "ext-appa-").
+4. Create a Google Cloud Console Project (e.g. `My IDPs External Groups Sync`).
+5. Add a Service Account to that Project.
+   - I recommend naming it after both the IDP you will use it for and the
+     application that needs the groups (e.g. `IDP 1 groups for App A`).
+6. Create a JSON Key for that Service Account.
+7. Share the Google Sheet that you created earlier with the `client_email` value
+   in that JSON Key file (as a Viewer, no notification).
+8. Set the following environment variables for your ID Broker instance:
+   - `EXTERNAL_GROUPS_SYNC_set1AppPrefix`
+     - Set this to some prefix starting with "ext-", e.g. `ext-appa`
+   - `EXTERNAL_GROUPS_SYNC_set1GoogleSheetId`
+     - Set this to the ID of the Google Sheet you created earlier.
+   - `EXTERNAL_GROUPS_SYNC_set1JsonAuthString`
+     - Use the JSON key you just created here, compacted to a single line by
+       something like this command:  
+       `cat service-account-key-from-google-abcdef123456.json | jq -c "."`
+9. You can also set the following environment variable if you want to send a
+   notification email any time the sync runs and encounters errors (such as
+   "No user found for email address ..." or "The given group (ext-appb-users)
+   does not start with the given prefix (ext-appa)"):
+   - `EXTERNAL_GROUPS_SYNC_set1ErrorsEmailRecipient`
+     - Set this to a single email address.
+10. If you need to sync those custom groups to another IDP...
+    - Ensure that IDP is also running a recent enough version of ID Broker.
```

Comment:
```suggestion
    - Ensure that IDP is also running ID Broker version 6.8.0 or later.
```

Reply:
Since I mentioned a specific version further up, I didn't want to duplicate that here (e.g. in case we later update the minimum version needed in one place and forget to update it in the other).